docs/elements.md

```diff
@@ -1891,6 +1891,59 @@ element contents only in a specific panel.
 
 ---
 
+### `pl-hidden-hints` element
+
+Display progressive hints that reveal themselves as the number of student submissions increases for the current variant.
+Hints must be opened by the user until they are revealed.
```

Comment:
'Hints must be opened by the user until they are revealed.' The meaning of this sentence is not clear to me. Are you trying to say something about when they are or aren't opened by default? It almost makes it sound like the user _must_ take some action on a hint, but they can just ignore if they want, right?